### DISCUSSION TITLE
Login Failures

### DISCUSSION THREAD

**Discussion by manthoszh** [FTBTeam/FTB-App#1141] [General]
Hey all, Just wondering if anyone knew anything specific about the login failures that seem to be happening to a lot of people. Errors where it won't recognize that you own Minecraft, can't connect to xbox services, and more. Seems there is a disconnect between Xbox/microsoft and this app and I'm not sure what side the problem actually lies. Any info and or workarounds would be great.

**Comment by MichaelHillcox** [comment]
From our Discord

## Auth issues with the App
Hi all, we've noticed an increase in users having issues logging into their Minecraft accounts using the FTB App. 

From our investigation, it looks like this is an issue on Mojangs side, and we hope they will have resolved this issue soon. We have confirmed this issue is happening in other Launchers as well.... 😢

The only known work around right now is to take a break and come back to the app in an hour or so and hope that it's been long enough that Mojang will no longer think you've spammed their servers (Even though you haven't)

Sorry for any inconvenience this may have caused ❤️

### Technical stuff
It looks like users are randomly getting 429 errors on any one of the Mojang API's. From user reports, our users aren't sending loads of requests, and it's just randomly happening... Due to this being a 429 error (Which means too many requests) we don't have any real workarounds other than giving it a break and coming back to it later...
